app/code/Magento/InventoryImportExport/Plugin/Import/SourceItemImporter.php

```diff
@@ -55,26 +57,35 @@ public function __construct(
     }
 
     /**
-     * Handle Import of Stock Item Data
+     * Plugin around Import to import Stock Data to Source Item
      *
+     * @param StockItemImporter $subject
+     * @param callable $proceed
      * @param array $stockData
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     * @throws \Magento\Framework\Exception\InputException
+     * @throws \Magento\Framework\Validation\ValidationException
      * @return void
+     * @see StockItemImporterInterface::import()
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function import(array $stockData)
-    {
+    public function aroundImport(
```

Comment:
looks like afterPlugin should be okay, for us.

It's better to avoid around plugins wherever its possible. They slow and potentially unsafe, because there is no guarantee that method being pluginized would be called

Reply:
To achieve this i've made the new CE implementation return the $stockData array parameter so that its available in the MSI afterPlugin, if not, we can't use afterPlugin because we wouldn't have access to the data array to import SourceItems